lms/static/js/instructor_dashboard/membership.js

```diff
@@ -42,7 +42,7 @@ such that the value can be defined later than this assignment (file load order).
                 add_btn_label: 'Add Member',
                 add_handler: function() {}
             });
-            templateHtml = window.Mustache.render($('#member-list-widget-template').html(), memberListParams);
+            templateHtml = _.template($('#membership-list-widget-tpl').html())(memberListParams);
```

Comment:
Yep, quality failure.

Build failed running pavelib.quality.run_xsslint: XSSLinter Failed.
20:26:06 Too many violations total (1853).
20:26:06 The limit is 1852.